.ci/prepare-settings.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+date=$(date +"%m%d%y")
+
+# Copying release-settings.xml
+mkdir -p .ci-temp
+cp ./.ci/release-settings.xml .ci-temp/
+
+# Putting values of environment variables in .ci-temp/release-settings.xml
+replace() {
+  sed -i "s/$1/$2/g" .ci-temp/release-settings.xml
+}
+
+replace SONATYPE_USER
+replace SONATYPE_PWD
+replace SOURCEFORGE_USER
+replace SOURCEFORGE_PWD
+replace GPG_PASSPHRASE
+replace GPG_KEY
+
+# Backup of settings.xml
+file1="./.ci/release-settings.xml"
+file2="~/.m2/settings.xml"
+
+if cmp -s "$file1" "$file2"; then
+  ~/.m2/settings.xml | sudo tee -a ~/.m2/settings.xml.backup."${date}" 2>&1
```

Comment:
Why not simple `mv`?

Reply:
`CMP` is comparing `~/.m2/settings.xml` and `./.ci/release-settings.xml` and if it found that there is content which is not related, then it is creating a backup file.

Reply:
Please explain why you can not simply rename file by `mv` command

Reply:
I can mv command, but first, we need to check if both the files are similar or not?

If they are similar then there is no need of creating a backup file. BUT, after comparing these files, if it founds any dissimilarities, then we can create a backup file.

Reply:
I have no problem with cmp. I am questioning single line of backup. Please use mv.

Reply:
Changes Done